resources/views/platform/components/game/achievements-list/type-indicator.blade.php

```diff
@@ -2,55 +2,78 @@
     'achievementType' => null,
     'beatenGameCreditDialogContext' => 's:|h:',
     'gameId' => null,
-    'progressionTypeValue' => 'progression', // `AchievementType`
-    'winConditionTypeValue' => 'win_condition', // `AchievementType`
     'isCreditDialogEnabled' => true,
 ])
 
 <?php
+use App\Platform\Enums\AchievementType;
 use Illuminate\Http\Request;
 
 $gameId ??= request()->route('game');
+
+$progressionTypeValue = AchievementType::Progression;
+$winConditionTypeValue = AchievementType::WinCondition;
+$missableTypeValue = AchievementType::Missable;
+
+$backgroundColorClassName = "bg-embed light:bg-neutral-50";
+if ($achievementType === $missableTypeValue) {
+    $backgroundColorClassName = "bg-amber-950 light:bg-amber-100";
+}
```

Comment:
My only reservation with the natural black color is I'm afraid it reduces how easy it is to skim and quickly recognize the beaten credit indicators. I do feel there probably should be some distinction -- maybe the reddish hue is too loud.

Reply:
I'm partially colorblind and for me the red makes it harder to see, it blends in with the background. The natural black stands out more in contrast. Try applying a black and white filter in an image editor to see what I'm seeing

Reply:
@Jamiras I've made a slight adjustment in latest. Now, instead of the reddish hue, the icon gets the standard embed color but the container has a subtle dashed outline. Let me know if you think this is any better.


https://github.com/RetroAchievements/RAWeb/assets/3984985/9af18a39-6f7d-4104-845f-a479a55e4fd5


Still would like some way to keep these visually distinct from the beaten credit indicators, just subtle enough to matter while skimming the page.

Reply:
I have no strong opinions for or against the dashed outline, but it's not showing up for me.
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/ee420e4b-f90a-40ad-94b7-c09291e7489c)

I ran `.hooks/post-merge` and `sail npm run build`. Is there something else I need to run?

Reply:
My bad - accidentally slipped in a bug right before I committed. It should be resolved now.